doc/design.txt

```diff
@@ -222,21 +218,31 @@ An image trailer has the following structure:
     |                       MAGIC (16 octets)                       |
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
-These records are at the end of each image slot.  The offset immediately
-following such a record represents the start of the next flash area.
+The offset immediately following such a record represents the start of the next
```

Comment:
Hmm. This sentence is still somewhat awkward, and I think you've removed the part that is easier to understand. The current wording sounds like a good way to describe it for a standard document, but not really a design document. How about something like:

"The image trailer is placed such that it butts up against the end of the partition. For example, if the following slot begins at offset 'n', the MAGIC field will be at offset 'n-16'."